app/scripts/check_requirements

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env php
+<?php
+
+// Script should not be called from the Web
+if (php_sapi_name() != 'cli') {
+    die('Nope');
+}
+
+/**
+ * Colorize the shell output
+ *
+ * @param string $text The text.
+ * @param string $color Color of the text: green, red or nothing.
+ * @return string
+*/
```

Comment:
Sorry to be such a pain in the ass, but consistency in the codebase is really important.

If you're documenting a function, the last * is aligned with the others. 

Let's keep the opening { on the same like of function/foreach, etc. Also unneeded indentation in the body of this function.